packages/roosterjs-editor-plugins/lib/plugins/TableResize/TableResize.ts

```diff
@@ -179,7 +196,7 @@ export default class TableResize implements EditorPlugin {
                                         verticalInserterTd,
                                         map.rect
                                     );
-                                    break;
+                                    return;
```

Comment:
I prefer to use break.

In the future if we want to add some other code after this loop, we may not notice that the code can return here and the new added code won't be executed.